src/components/inventory/GeneralInfo/TextInputModal/TextInputModal.js

```diff
@@ -37,20 +39,27 @@ export default class TextInputModal extends Component {
                 title={ title }
                 className="ins-c-inventory__detail--edit"
                 aria-label={ ariaLabel ? `${ariaLabel} - modal` : 'input modal' }
+                ouiaId={ modalOuiaId }
                 isOpen={ isOpen }
                 onClose={ event => onCancel(event) }
                 actions={ [
-                    <Button key="cancel" data-action="cancel" variant="secondary" onClick={ onCancel }>
+                    <Button key="cancel" data-action="cancel" variant="secondary" onClick={ onCancel } ouiaId={ cancelOuiaId }>
                         Cancel
                     </Button>,
-                    <Button key="confirm" data-action="confirm" variant="primary" onClick={ () => onSubmit(this.state.value) }>
+                    <Button
+                        key="confirm"
+                        data-action="confirm"
+                        variant="primary"
+                        onClick={ () => onSubmit(this.state.value) } ouiaId={ confirmOuiaId }
```

Comment:
Spacing is a bit off in here
```suggestion
                        onClick={ () => onSubmit(this.state.value) }
                        ouiaId={ confirmOuiaId }
```